### ISSUE TITLE
[Bug]: .get() is not idempotent

### ISSUE THREAD

**Issue by azaroth42** [project-lux/luxy#2]
### Priority Level

High

### What happened?

Calling get multiple times on the same context causes a traceback, and should probably be idempotent?

### Relevant log output

```shell
>>> from luxy import Objects
>>> r = Objects().filter(classification={"name": "painting"})
>>> res = r.get()
>>> res2 = r.get()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/rs2668/Development/lux/luxy/luxy/api.py", line 130, in get
    self.num_results = self.num_results(self.json)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'int' object is not callable
```

### Additional Tasks

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

**Comment by wjbmattingly** [comment]
Thanks! Fixed in 0.0.2 and will be merged into main soon.

**Comment by wjbmattingly** [comment]
Closed with #4
